raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/PeerIdentityValidator.java

```diff
@@ -0,0 +1,10 @@
+package org.hypertrace.core.rawspansgrouper;
+
+import org.hypertrace.core.spannormalizer.PeerIdentity;
+
+public class PeerIdentityValidator {
+
+  public static boolean isValid(PeerIdentity peerIdentity) {
+    return IpIdentityValidator.isValid(peerIdentity.getIpIdentity());
```

Comment:
is there a function called peerIdentity.hasIpIdentity? if yes we should check that the IP identity is available